network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -24,6 +24,12 @@ class QueryAnswer:
     numerical_result: int = 0
 
 
+@dataclass
+class QueryOutput:
+    res: int = 0
+    query_output: str = ''
```

Comment:
Do we really need a new class? I find it confusing.
Maybe we can just add `query_output` field to `QueryAnswer`...
`res` seems to be always equal to `QueryAnswer.numerical_result`